change_of_vars_demo1d/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        latexify(width_scale_factor=2,fig_height=1.5)
</pre><p>We should use width_scale_factor=1 because we need 6 inch wide figure</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>